site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/model/SiteAttributeConfiguration.java

```diff
@@ -9,6 +9,7 @@
 public class SiteAttributeConfiguration {
 
     private String name;
+    private String type;
```

Comment:
What is "type" here? If it is a class name then better to use a `Class<?>` reference

Reply:
I guess this is the type of the field, string in all cases till now but since we also have an address, @shajidissa had to add this. right?

Reply:
Yes, type is now needed to differentiate between string and complex objects like address.